src/OrchardCore/Orchard.Environment.Extensions/ExtensionManager.cs

```diff
@@ -90,6 +90,11 @@ public class ExtensionManager : IExtensionManager
 
         public IExtensionInfo GetExtension(string extensionId)
         {
+            if (string.IsNullOrEmpty(extensionId))
```

Comment:
Can you provide an explanation on how you reach this state?
Depending on your answer we might prefer to throw an exception instead.